src/model/messages/mrtd/EMrtdDataSubmitMessage.ts

```diff
@@ -81,6 +81,7 @@ export class EMrtdDataSubmitMessage extends BaseMessage {
       }
     })(mrzString.length)
     const parsedMrz = Mrz.parse(formattedMrz)
+    const birthDateFromNfc = parsed.fields.additionalPersonalData?.fullDateOfBirth
```

Comment:
```suggestion
    const birthDateFromAdditionalPersonalData = parsed.fields.additionalPersonalData?.fullDateOfBirth
```

Because both the MRZ and the additionalPersonalData come from 'the NFC' (both are encoded in Elementary Files and read over NFC).